process-migration-service/pom.xml

```diff
@@ -113,6 +125,12 @@
     <dependency>
       <groupId>io.thorntail</groupId>
       <artifactId>microprofile-config</artifactId>
+      <exclusions>
+        <exclusion>
+          <artifactId>narayana-jts-idlj</artifactId>
+          <groupId>org.jboss.narayana.jts</groupId>
```

Comment:
I checked with guys from Thorntail. This is not needed at all, so the exclusion is fine.